### PR TITLE
Reducers actions

### PR DESCRIPTION
**Pull request by anita00001**
## In this branch, we have:
- [ ] Added Redux Toolkit (`npm install react-redux @reduxjs/toolkit`)
- [ ] Created a directory that will contain all Redux logic (`/src/redux`)
- [ ] Configured a Redux store (`/src/redux/store.js`)
- [ ] Defined a `slice` of state for `books` that:
    - Includes **an array of books** (initial state: empty array)
    - Includes a **reducer that adds a book**
    - Includes a **reducer that removes a book**
- [ ] Defined a `slice` of state for `categories` that:
    - Includes an a**rray of categories** (initial state: empty array)
    - Includes a **reducer** that checks the `status` and always returns "**Under construction**" (the initial state should check to that string)

## Key Feature:
- [ ] Structure application files in [ducks pattern](https://github.com/erikras/ducks-modular-redux) for your Redux files. Your files/folder structure could look like this:
```
/src
    |---/components
    |---/redux
        |--- /books
                | booksSlice.js
        |--- /categories
                | categoriesSlice.js
        | store.js
    | index.js
```

### CODE REVIEW

File: src/redux/store.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { configureStore } from '@reduxjs/toolkit';
+// import booksReducer from './books/booksSlice';
+// import categoriesReducer from './categories/categoriesSlice';
+
+const store = configureStore({
+  reducer: {
+    // books: booksReducer,
+    // categories: categoriesReducer,
```
Comment (codepantha) [inline src/redux/store.js]: - [ ] Please import `books` and `categories` and add them to the to `configureStore` method.
### HINT:
Please remove the comment symbols from the lines of code that have been commented out. 😎

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @anita00001 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ DescriptiveP PR
✔️ Configured Redux store 📌 
✔️ Actions and reducers for adding and removing books 📌 
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/store.js (hunk 1 above)
Comment (anita00001) [reply src/redux/store.js]: I tried to import booksReducer and categoriesReducer, but faced an issue:
- The ESlint passes on my local but fails on GitHub
ESLint passing on my terminal
image
ESLint failing on GitHub:
image

I tried adding the file extension as **.js** for both imports (`./books/booksSlice.js` and `./categories/categoriesSlice.js`), But it fails to throw error: "Unexpected use of file extension **"js"** "  both on my terminal and on GitHub
ESLint failing on my terminal when the **.js** extension has been used:
image

@codepantha

**Review by rustam-tolipov — APPROVED** [review]
Hi @anita00001, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯
3. Well-implemented code based on previous reviewer's request 👍 

## Optional suggestions
No optional suggestions :)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**